shared/src/testFixtures/kotlin/com/egm/stellio/shared/util/NgsiLdTestUtils.kt

```diff
@@ -1,36 +1,32 @@
 package com.egm.stellio.shared.util
 
-import com.egm.stellio.shared.model.NgsiLdGeoProperty
+import com.egm.stellio.shared.model.JsonLdEntity
 import com.egm.stellio.shared.model.QueryParams
-import com.egm.stellio.shared.model.parseToNgsiLdAttributes
 
-fun parseGeoFragmentToPointGeoProperty(
+fun gimmeSimpleEntityWithGeoProperty(
     propertyKey: String,
     longitude: Double,
     latitude: Double
-): NgsiLdGeoProperty {
+): JsonLdEntity {
     val locationFragment =
```

Comment:
it's not anymore a fragment of the location but rather an entity with just the location attribute now

Reply:
indeed